test/Number/toString.js

```diff
@@ -9,6 +9,7 @@ runTest('new Number("444" + "123.789123456789875436")');
 runTest('new Number(-444123.78963636363636363636)');
 runTest('new Number(0)');
 runTest('1e21');
+runTest('1.1e21');

```

Comment:
Maybe also add tests for `1e21+1` and `1e21-1` (integers one before and after the boundary) and compare output with other engines. Maybe `1e21+0.1` and `1e21-0.1` also?

Reply:
These numbers all equal to `1e21` due to the loss of precision. How about adding `0.9999999999999999e21` and `1.0000000000000001e21`?

Reply:
Yeah, that sounds good. :)